### ISSUE TITLE
DTS CI fails on cloning `dts-release-cicd-pipeline`

### ISSUE THREAD

**Issue by PLangowski** [Dasharo/dasharo-issues#1052]
### Component

Dasharo Tools Suite

### Device

other

### Dasharo Tools Suite version

v2.0.0-rc*

### Brief summary

DTS CI fails to clone `dts-release-cicd-pipeline`: Permission denied

### How to reproduce

1. Push a release tag to develop

### Expected behavior

CI passes. The release is published on GitHub and binaries + signatures are deployed.

### Actual behavior

`Trigger signing` job fails:

```
Cloning into 'dts-release-cicd-pipeline'...
[main 84ab3340d1c7] Signing release v2.0.0-rc6
 1 file changed, 1 insertion(+), 1 deletion(-)
git@git.3mdeb.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

The release is not published.

### Additional context

This problem has appeared in the past and seemed to have been fixed in https://github.com/Dasharo/meta-dts/pull/146, however it reappeared.
The `SSH_KEY_CI_CD` secret should hold the correct key that will allow the builder to access https://git.3mdeb.com/3mdeb/dts-release-cicd-pipeline.

**Comment by m-iwanicki** [comment]
@PLangowski probably not the fault but there is open PR to merge changes that were merged to main but not into develop: https://github.com/Dasharo/meta-dts/pull/147
Maybe there were other fixes that were applied to main but not develop?

**Comment by m-iwanicki** [comment]
Not sure if it's connected but last `Weekly clean & rebuild` failed during `Send sstate-cache folder to cache.dasharo.com` step:

```
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@    WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY!
Someone could be eavesdropping on you right now (man-in-the-middle attack)!
(...)
```

**Comment by m-iwanicki** [comment]
Should be fixed by https://github.com/Dasharo/meta-dts/pull/173 but needs to be tested.
Workflow failed a little later not during clone but most likely push (logs didn't show which command failed)

**Comment by artur-rs** [comment]
@m-iwanicki merged. let's retest the ci with the DTS optiplex changes

**Comment by artur-rs** [comment]
Closing. We've successfully released the v2.0.0, and v2.0.1-RCs version so far: https://github.com/Dasharo/meta-dts/releases
